config/clusters/qcl/common.values.yaml

```diff
@@ -48,13 +43,6 @@ jupyterhub:
           - gizmo404
           - jtkmckenna
   singleuser:
-    image:
-      # pangeo/pangeo-notebook is maintained at: https://github.com/pangeo-data/pangeo-docker-images
-      name: pangeo/pangeo-notebook
-      # pullPolicy set to "Always" because we use the changing over time tag
-      # "latest".
-      pullPolicy: Always
-      tag: "latest"
```

Comment:
QCL are using `latest` tags for their configured images, but pullPolicy defaults to the `Always` behavior when unspecified and the tag is `latest` anyhow.

Reply:
We just discovered that this is actually not true, due to a kubespawner bug. https://github.com/jupyterhub/kubespawner/pull/807 has details.

Reply:
https://github.com/2i2c-org/infrastructure/pull/3396 temporarily sets imagePullPolicy again